docs/sources/mimir/operators-guide/run-production-environment/scaling-out.md

```diff
@@ -136,7 +136,12 @@ The required amount of time to wait depends on your configuration and it's the m
 
 To guarantee no downtime when scaling down [store-gateways]({{< relref "../architecture/components/store-gateway.md" >}}), complete the following steps:
 
-1. Scale down no more than two store-gateways at the same time.
 1. Ensure at least `-store-gateway.sharding-ring.replication-factor` store-gateway instances are running (three when running Grafana Mimir with the default configuration).
-
-> **Note:** If you enabled [zone-aware replication]({{< relref "../configure/configure-zone-aware-replication.md" >}}) for store-gateways, you can in parallel scale down any number of store-gateway instances in one zone at a time.
+1. Scale down no more than two store-gateways at the same time. If you enabled [zone-aware replication]({{< relref "../configure/configure-zone-aware-replication.md" >}})
+   for store-gateways, you can in parallel scale down any number of store-gateway instances in one zone at a time. Zone-aware replication is enabled by default in the `mimir-distributed` Helm chart.
+1. Stop the store-gateway instances you want to scale down.
+1. Remove the stopped instances from the store-gateway ring. In your browser go to the `GET /store-gateway/ring` page that store-gateways expose on their HTTP port
```

Comment:
This depends on this setting:

```
# Unregister from the ring upon clean shutdown.
  # CLI flag: -store-gateway.sharding-ring.unregister-on-shutdown
  [unregister_on_shutdown: <boolean> | default = true]
```

With the default config, it unregisters from the ring on shutdown so there's no action to take. I would clarify it.

Reply:
i think i addressed this in https://github.com/grafana/mimir/pull/3698/commits/d8b3b96a738fb51838bfdb205d881349462cc103